builtin/common/metatable.lua

```diff
@@ -1,6 +1,6 @@
 -- Registered metatables, used by the C++ packer
 local known_metatables = {}
-function core.register_async_metatable(name, mt)
+function core.register_portable_metatable(name, mt)
```

Comment:
my only concern: does it make sense to alias this function for existing mods written against 5.9.0-dev?

Reply:
That would make sense.